atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1642,9 +1642,18 @@ private void putMetadataForTablesInternal(
     }
 
     private void putMetadataWithoutChangingSettings(final TableReference tableRef, final byte[] meta) {
```

Comment:
Yeah, this could be cleaned up. It's all a legacy of a shittier age that I've half moved us off of a while ago by making createTable also take in a table metadata byte[].

The ideal world also will remove kvs.putMetadataForTable(s) completely and make the guarantee explicit in the docs that createTable also can also be 'update table to new settings' if a table already exists.